apps/web/components/team/screens/Team.tsx

```diff
@@ -32,7 +32,7 @@ const Member = ({ member, teamName }: { member: MemberType; teamName: string | n
             {!isBioEmpty ? (
               <>
                 <div
-                  className="dark:text-darkgray-600 text-s text-gray-500"
+                  className="dark:text-darkgray-600 text-sm text-gray-500 [&_a]:text-blue-500 [&_a]:underline [&_a]:hover:text-blue-600"
                   dangerouslySetInnerHTML={{ __html: md.render(member.bio || "") }}
                 />
```

Comment:
this is quite repetitive. mostly because the profile page is not a component yet and the code is rewritten in all places.

we should make a `<Profile>` component or something similar that covers `<Profile team={teamID} />` too